### PR TITLE
Travis - change matrix order for faster results

### PR DESCRIPTION
**Pull request by gharlan**

### CODE REVIEW

**Comment by gharlan** [comment]
if ordering by version is unimportant, we could use this matrix (fastest and most interesting builds at top):

```yml
matrix:
    fast_finish: true
    include:
        - php: hhvm
        - php: 7.0
        - php: 7.0
          env: SYMFONY_VERSION="~2.8"
        - php: 5.3
          env: 'COMPOSER_FLAGS="--prefer-stable --prefer-lowest"'
        - php: 5.4
        - php: 5.5
        - php: 5.6
```

**Comment by keradus** [comment]
putting slowest jobs at the end is nightmare - it causes whole build to be slow.

**Comment by gharlan** [comment]
My main target: get faster result when a build is failing, because then I have to change something in my pr.
And a second target: get a faster tendency. When hhvm, 7.0 and 5.3 are passing, there is a good chance that whole build will pass. So I prefer my second proposal.

**Comment by gharlan** [comment]
Or this one:

```yml
matrix:
    fast_finish: true
    include:
        - php: 5.3
          env: 'COMPOSER_FLAGS="--prefer-stable --prefer-lowest"'
        - php: 7.0
        - php: hhvm
        - php: 5.4
        - php: 5.5
        - php: 5.6
        - php: 7.0
          env: SYMFONY_VERSION="~2.8"
```

**Comment by GrahamCampbell** [comment]
I don't think what we have needs changing.

**Comment by gharlan** [comment]
Ok. @keradus your comment was a :-1:, right?
Then we can close it ;)

**Comment by keradus** [comment]
I'm :-1: for flip-flop the order as you done in commit.
But I'm **not** :-1: for changing the order, eg only move 7.0 on top (but I'm not :+1: for that neither, just :±0: ;) )

**Comment by gharlan** [comment]
(updated to my current proposal..)

any other thoughts?

**Comment by SpacePossum** [comment]
lgtm :+1:

File: .travis.yml [hunk 1]
```diff
@@ -5,14 +5,13 @@ matrix:
     include:
         - php: 5.3
           env: 'COMPOSER_FLAGS="--prefer-stable --prefer-lowest"'
+        - php: 7.0
```
Comment (keradus) [inline .travis.yml]: why removing the `SYMFONY_VERSION="~3.0"` ?

Comment (gharlan) [inline .travis.yml]: imo it is useless

Comment (gharlan) [inline .travis.yml]: later I would add `LINT_TEST_CASES=1` here.

**Comment by keradus** [comment]
Thank you @gharlan.

**Pull request merged**